### PR TITLE
Build using io.elementary.Loki.BaseApp

### PR DESCRIPTION
**Pull request by bilelmoussaoui**
Reduces the number of libraries and includes the default gtk theme of elementary.

### CODE REVIEW

**Comment by Alexander-Wilms** [comment]
bot, build

**Comment by Alexander-Wilms** [comment]
Thanks, merged

File: com.github.dahenson.agenda.json [hunk 1]
```diff
@@ -19,56 +20,7 @@
 		"cflags": "-O2",
 		"cxxflags": "-O2"
 	},
-	"cleanup": [
```
Comment (TingPing) [inline com.github.dahenson.agenda.json:22]: You still need some of the cleanup

Comment (Alexander-Wilms) [reply com.github.dahenson.agenda.json]: Hm I tried adding
```
	"cleanup": [
		"/bin/vala*",
		"/bin/vapi*",
		"/include",
		"/lib/pkgconfig",
		"/share/vala",
		"/share/vala-0.36",
		"*.a",
		"*.la"
	],
```
but nothing was deleted during the cleanup stage. Will check if that happens with other manifests as well.

**Pull request merged**